src/style/style_layer/fill_style_layer.js

```diff
@@ -24,6 +24,7 @@ class FillStyleLayer extends StyleLayer {
     }
 
     recalculate(parameters: EvaluationParameters) {
+        super.recalculate(parameters);
         this.paint = this._transitioningPaint.possiblyEvaluate(parameters);
```

Comment:
This already happens in the base class's `recalculate`, which we're now calling.